### DISCUSSION TITLE
errors with Locale and intl

### DISCUSSION THREAD

**Discussion by darkogav** [shivammathur/homebrew-php#2506] [Q&A / Help]
Hi, we have an application that works fine on php 7.4 and php 8.3 on other development platforms. I am working on getting it to work on mac os with homebrew using the shivammathur brew tap. 

I keep getting the error 

> "Cannot declare class Locale, because the name is in use.. "

I checked the extensions and modules in httpd and all are same as what is currently working on other platforms. but we keep getting this error and I suspect it's related to intl extension. or maybe another extension. 

Is there a correct way to disable and enabled the intl extensions? Editing the php.ini in/opt/hombrew/etc/php/8.3/php.ini  and restarting httpd and php@8.3 services  does not seem to have an effect.

**Comment by shivammathur** [comment]
@darkogav 

intl extension is compiled with php statically, so you cannot disable that in Homebrew's php builds.

I would suggest namespacing your classes to avoid this.
https://www.php.net/manual/en/language.namespaces.php

**Reply by darkogav** [reply]
thank you.
